src/app/models/wizard.ts

```diff
@@ -0,0 +1,233 @@
+//TODO: refactor and refine domain model for wizard and configurator
+import { Space } from './space'
+
+export class Wizard {
```

Comment:
yes i figured ... i still wanted to rename/refactor/remove some classes and I have some better names for a more refined and clearer abstraction so just put it all in the wizard bucket untill ...